themes/darcula.color-theme.json

```diff
@@ -251,25 +277,31 @@
             "scope": [
                 "keyword.operator.combinator.css",
                 "keyword.operator.pattern.css",
+                "keyword.operator.pattern.scss",
                 "meta.method.body.java variable.other.definition.java",
                 "meta.template.expression.js",
                 "meta.template.expression.ts",
                 "meta.template.expression.tsx",
                 "punctuation.definition.entity.begin.bracket.square.css",
-                "punctuation.definition.entity.css",
+                "punctuation.definition.attribute-selector.begin.bracket.square.scss",
+                "punctuation.definition.entity.css", // Used for css and scss.
```

Comment:
These comments can all be removed, after removing them I'll merge this PR :+1:

Reply:
Done, thank you.